sos/report/plugins/foreman_proxy.py

```diff
@@ -0,0 +1,123 @@
+# Copyright (C) 2021 Red Hat, Inc., Pavel Moravec <pmoravec@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import (Plugin, RedHatPlugin, DebianPlugin,
+                                UbuntuPlugin)
+
+
+class ForemanProxy(Plugin):
+
+    short_desc = 'Foreman SmartProxy systems management'
+
+    plugin_name = 'foreman_proxy'
+    profiles = ('sysmgmt',)
+    packages = ('foreman-proxy', )
+
+    option_list = [
+        ('proxyfeatures', 'list features of the smart proxy', 'slow', False),
+    ]
+
+    def setup(self):
+        self.add_forbidden_path([
+            "/etc/foreman-proxy/*key.pem",
+        ])
+
+        _hostname = self.exec_cmd('hostname')['output']
+        _hostname = _hostname.strip()
+        self.add_copy_spec([
+            "/etc/foreman-proxy/",
+            "/etc/smart_proxy_dynflow_core/settings.yml",
+            "/etc/foreman-installer/",
+            "/var/log/foreman-proxy/cron*log*",
+            "/var/log/foreman-proxy/migrate_settings*log*",
+            "/var/log/foreman-proxy/proxy*log*",
+            "/var/log/foreman-proxy/smart_proxy_dynflow_core*log*",
```

Comment:
Perhaps it should just capture 
```suggestion
            "/var/log/foreman-proxy/*log*",
```

Reply:
We might risk that a huge file content (there were such use cases) will cause skipping some other logfile type due to 25MB or 100MB default sizelimit. On the other side, such a risk is small and we migh skip collecting some other logfiles with different pattern that might appear there.  So ACK.